File: Csharp/Exes and ohs.cs
// https://www.codewars.com/kata/55908aad6620c066bc00002a/train/csharp
using System.Linq;
using System;
public static class Kata 
{
  public static bool XO (string input)
  {
     return input.ToLower().Count(i => i == 'x') == input.ToLower().Count(i => i == 'o');
  }
}

File: javascript/Exes and ohs.js
// https://www.codewars.com/kata/55908aad6620c066bc00002a/train/javascript?
function XO(str) {
    let n = str.toLowerCase().split('')
    return n.filter(e=>e === "x" ).length === n.filter(e=>e === "o").length ? true : false
    
    
}